JS/Lesson22/savas/index.html

```diff
@@ -3,9 +3,37 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
   <title>Accordion</title>
 </head>
 <body>
-  
+  <h1>2023</h1>
+  <h2>BEST GAMING CONSOLES</h2>
+  <h3>Choose your game console from <span>the quick list:</span></h3>
+  <div class="accordion">
+
+      <div class="boxes">
+        <div class="label">Play Station 5</div>
```

Comment:
I'd name this class 'accordion-header' to describe the purpose of the element better